dietpi/dietpi-software

```diff
@@ -4483,11 +4483,11 @@ _EOF_
 
 			Banner_Installing
 
-			INSTALL_URL_ADDRESS='https://dietpi.com/downloads/binaries/all/Ubooquity.jar'
-			G_CHECK_URL "$INSTALL_URL_ADDRESS"
-
-			mkdir -p $G_FP_DIETPI_USERDATA/ubooquity
-			G_EXEC wget "$INSTALL_URL_ADDRESS" -O $G_FP_DIETPI_USERDATA/ubooquity/Ubooquity.jar
+			G_EXEC curl -sSfL https://vaemendis.net/ubooquity/service/download.php -o Ubooquity.zip
+			G_EXEC unzip -o Ubooquity.zip
+			G_EXEC_NOEXIT=1 G_EXEC rm Ubooquity.zip
+			G_EXEC mkdir -p $G_FP_DIETPI_USERDATA/ubooquity
+			G_EXEC mv {,$G_FP_DIETPI_USERDATA/ubooquity/}Ubooquity.tar 
```

Comment:
is this line correct. I guess it's a *.jar file and not a *.tar one?

Reply:
Good find!